exonum_client/api.py

```diff
@@ -274,6 +289,86 @@ def available_services(self) -> requests.Response:
         """
         return self.get(self._system_url.format("services"))
 
+    def get_instance_id_by_name(self, name: str) -> Optional[int]:
+        """
+        Gets an ID of the service instance with the provided name.
+
+        Example:
+
+        >>> public_api = PublicApi("127.0.0.1", 80)
+        >>> id = public_api.get_instance_id_by_name("cryptocurrency")
+        >>> id
+        42
+
+        Parameters
+        ----------
+        name: Name of the service.
+
+        Returns
+        -------
+        result: int
```

Comment:
Nit: should be `Optional[int]` and please add a note about conditions in which `None` is returned.

Reply:
Done